distribution/docs/src/main/resources/content/_tables/Session.adoc

```diff
@@ -19,7 +19,10 @@
 |Session Timeout (in minutes)
 |expirationTime
 |Integer
-|The number of minutes after a session has been inactive that it should be invalidated.
+|Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 31 minutes.
```

Comment:
Thanks for the update! Does this timeout only affect the UIs? Are there other places in which you could be logged out automatically (e.g. SSH)?

Reply:
The SSH timeout is a different configuration: https://karaf.apache.org/manual/latest/remote.

In our version of the config there is no property set for it: https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/org.apache.karaf.shell.cfg.